src/components/transaction/DataFormat.vue

```diff
@@ -29,7 +29,7 @@ export default defineComponent({
         const transferData = computed(() => actionData.value as TransferData);
         const clientHeight = computed(() => dataBox.value?.clientHeight ?? 0);
         let currentData = ref<string | unknown>(null);
-        const maxHeight = 65; // the maximum row height
+        const maxHeight = 98; // the maximum row height, should this be scalar (3) * constant (line height)?
```

Comment:
```suggestion
        const maxHeight = 98; // the maximum row height
```
In this case it would add an additional variable that is only used once as maxHeight is referenced elsewhere